src/Uno.UWP/ApplicationModel/Contacts/ContactQuerySearchFields.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Windows.ApplicationModel.Contacts
+{
+	[Flags]
+	public enum ContactQuerySearchFields

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the 'FlagsAttribute' from this enum.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4900834)

Reply:
I think removing [flags] would lead to error. Probably it complains about -1, but -1 is all ones.
See https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.contacts.contactquerysearchfields .

Reply:
False positive.  I'll remove this analysis, it's misleading.